lib/mmonit/connection.rb

```diff
@@ -40,7 +40,11 @@ def connect
 		end
 
 		def login
-			self.request('/z_security_check', "z_username=#{@username}&z_password=#{@password}", true).code.to_i == 302
+			self.request('/z_security_check', URI.encode_www_form({'z_username'=>@username, 'z_password'=>@password, 'z_csrf_protection'=>'off'}), true).code.to_i == 302
+		end
+
+		def logout
```

Comment:
And you're right, the logout method invalidates the zsessionid cookie retrieved in 'connect', so it should wipe @http to force a re-connect. Maybe we should call it 'disconnect' instead?